### PR TITLE
Added setAuthenticated when users verify their MFA for the first time

### PR DESCRIPTION
**Pull request by alexmylonas**
## Background

- First time users had to re-enter email/pass and MFA the first time the login in UI

This PR resolves this #142 

## Changes

- Added authenticated as true when a first time users MFA is verified

## Testing

- Locally

### CODE REVIEW

File: web/src/components/utils/auth-context.tsx [hunk 1]
```diff
@@ -281,10 +281,10 @@ const AuthProvider: React.FC = ({ children }) => {
       try {
         await Auth.verifyTotpToken(authUser, mfaCode);
         await Auth.setPreferredMFA(authUser, 'TOTP');
-
-        const userWithMFA = await Auth.currentAuthenticatedUser();
-        setAuthUser(userWithMFA);
-
+        // NOTE: User is confirmed at this point so we can go ahead an logged in here
```
Comment (3nvi) [inline web/src/components/utils/auth-context.tsx]: minor typo in `and`

Comment (alexmylonas) [reply web/src/components/utils/auth-context.tsx]: Updated ;)

**Comment by kostaspap** [comment]
🎉

**Comment by austinbyers** [comment]
Congrats on your first Panther contribution, @alexmylonas !!! 🎉 🙌 

For future reference, if in the PR description you say `Closes: #(issueNumber)`, Github will underline it and link the associated issue

**Pull request merged**